autosfw2jpg/autosfw2jpg-only-sfws

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+BLOCK_DEVICE=/dev/sr0
+ROLL_NUMBER=$1
+PICTURE_DATE=$2
+
+#clear workspace
+rm -f ~/tmp/conversions/*
+
+#Copy images from disk
+udisksctl mount -b $BLOCK_DEVICE
+let i=0
+for x in /media/dean/$ROLL_NUMBER/$ROLL_NUMBER.rol/*
```

Comment:
I suggest we make `/media/dean/` a variable `$DISK` as with the other script.

```bash
for x in $DISK/$ROLL_NUMBER/$ROLL_NUMBER.rol/*
```

What do you think?